task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter A Number :");
+            int inputNumber = int.Parse(Console.ReadLine());
+
+            if (inputNumber % 2 == 0 && inputNumber % 5 == 0)
+            {
+                Console.WriteLine("Tutti-Frutti");
+            }
+            else if (inputNumber % 2 == 0)
+            {
+                Console.WriteLine("Tutti");
+            }
+            else if (inputNumber % 5 == 0)
+            {
+                Console.WriteLine("Frutti");
+            }
+            else 
+                //(!(inputNumber % 2 == 0) && !(inputNumber % 5 == 0))
```

Comment:
We don't need this comment here. It is understandable from the code. We will come to this block only in one case. If number is not divided by 2 and 5. But if it is necessary for you you could leave this comment.